src/main/groovy/com/avast/gradle/dockercompose/tasks/ComposeDown.groovy

```diff
@@ -37,10 +37,17 @@ class ComposeDown extends DefaultTask {
     VersionNumber getDockerComposeVersion() {
         new ByteArrayOutputStream().withStream { os ->
             project.exec { ExecSpec e ->
-                e.commandLine 'docker-compose', '--version'
+                e.commandLine prepareCommand(['docker-compose', '--version'])
                 e.standardOutput = os
             }
             VersionNumber.parse(os.toString().trim().findAll(/(\d+\.){2}(\d+)/).head())
         }
     }
+
+    protected Iterable<String> prepareCommand(List<String> baseCommand) {

```

Comment:
Yes, it is required, otherwise you'll get error "_Can't find a suitable configuration file in this directory or any parent. Are you in the right directory?_". When using the -f option you can specify any filename and are not bound by 'docker-compose.yml' standard.